src/scheduler/server_info.c

```diff
@@ -3410,14 +3410,17 @@ read_formula(void)
 	char buf[RF_BUFSIZE];
 	size_t bufsize = RF_BUFSIZE;
 	int len;
-	char pathbuf[MAXPATHLEN];
+	char pathbuf[MAXPATHLEN + 100];
 	FILE *fp;
+	char cwd[MAXPATHLEN];
 
+	if (getcwd(cwd, sizeof(cwd)) == NULL) {
+		log_event(PBSEVENT_SYSTEM, PBS_EVENTCLASS_REQUEST, LOG_INFO, __func__,
+				"Error getting current working directory path");
+		return NULL;
+	}
 
-	if (dflt_sched)
-		sprintf(pathbuf, "%s/%s", pbs_conf.pbs_home_path, FORMULA_ATTR_PATH_SCHED);
-	else
-		sprintf(pathbuf, "%s/sched_priv_%s/%s", pbs_conf.pbs_home_path, sc_name, FORMULA_FILENAME);
+	snprintf(pathbuf, sizeof(pathbuf), "%s/%s", cwd, FORMULA_FILENAME);
```

Comment:
Why bother getting the cwd?  Why not just open FORMULA_FILENAME.  It will open it in the current directory.